build/images/ovs/Dockerfile.ubi

```diff
@@ -32,7 +32,7 @@ RUN cd /tmp/openvswitch* && \
     sed -e "s/@VERSION@/$OVS_VERSION/" rhel/openvswitch-fedora.spec.in > /tmp/ovs.spec && \
     yum-builddep -y /tmp/ovs.spec && ./boot.sh && \
     ./configure --prefix=/usr --localstatedir=/var --sysconfdir=/etc && \
-    make rpm-fedora && mkdir -p /tmp/ovs-rpms && \
+    RPMBUILD_OPT="--without libcapng" make rpm-fedora && mkdir -p /tmp/ovs-rpms && \
```

Comment:
Is there a public reference that documents how it fixes the issue? And better to add a comment as it can't be easily seen from the code itself.
Ideally the comment should also the other impacts of disabling libcapng if there are any.

Reply:
We don't set `--user` option when running OVS, right? If so, I think we could just set `--without libcapng`. We should add comment for reference in the future, including the link, the purpose, and the side effect.

Reply:
We did not use `--user` option ( `--ovs-user` option of ovs-ctl in `start_ovs`) in Antrea. 
I will add a comment for this change.